model/connectivity/src/main/java/org/eclipse/ditto/model/connectivity/ClientCertificateCredentials.java

```diff
@@ -50,7 +50,7 @@ private ClientCertificateCredentials(@Nullable final String clientCertificate,
 
     @Override
     public <T> T accept(final CredentialsVisitor<T> visitor) {
-        return visitor.clientCertificate(this);
+        return visitor.get(this);
```

Comment:
Hm, as this `get(this)` is `@Nullable` - wouldn't that break the contract of `Credentials.accept(CredentialsVisitor)` which is `@NonNull`?

So instead of this, should we still use the `clientCertificate(this)` and "un-deprecate"?